Documentation/operations/upgrade.rst

```diff
@@ -386,6 +386,8 @@ Changed Metrics
   met (e.g: NodePort service implementation in BPF is disabled or socket load-balancing
   is disabled), it will fail to initialize and will log an error instead of silently
   fall back to iptables based masquerading.
+* The ``--output`` flag now can be used with ``cilium-dbg encrypt status`` command to
+  return output in JSON or YAML format.
```

Comment:
I'm not sure if this is appropriate for the upgrade nodes. The upgrade notes are for letting users know about specific things they should keep in mind when upgrading to a new version of Cilium, in order to avoid unexpectedly breaking their install. This comment here feels more like a release note that we would put in a change log. Can we remove it?

Reply:
Nope, nothing is needed for upgrade notes for this change.

Reply:
Removed.